hub/management/commands/import_save_the_children_supporter_data.py

```diff
@@ -0,0 +1,51 @@
+from django.conf import settings
+
+import pandas as pd
+
+from hub.models import AreaData, DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand
+
+
+class Command(BaseImportFromDataFrameCommand):
+    help = (
+        "Import data about the number of Save the Children supporters per constituency"
+    )
+
+    message = "importing Save the Children shop count"
+    uses_gss = False
+    cons_row = "CONSTITUENCY"
+    data_file = (
+        settings.BASE_DIR / "data" / "2022_Campaigning_Actions_By_Constituency 1.csv"
+    )
+
+    defaults = {
+        "label": "Number of Save the Children supporters",
+        "description": "The number of Save The Children supporters per constituency.",
+        "data_type": "integer",
+        "category": "movement",
+        "subcategory": "supporters_and_activists",
+        "source_label": "Data from Save the Children.",
+        # "release_date": "2022",
+        "source": "https://www.savethechildren.org.uk/",
+        "source_type": "csv",
+        "data_url": "",
+        "table": "areadata",
+        "default_value": 10,
+        "comparators": DataSet.numerical_comparators(),
```

Comment:
Ideally we should have the unit_type and unit_distribution in here given it won't get backfilled by a migration.